### ISSUE TITLE
Add Support for Azure `azurerm_servicebus_namespace`

### ISSUE THREAD

**Issue by giladdi-tr** [infracost/infracost#1432]
Currently, infracost does not calculate the cost for this resource type.

**Comment by tim775** [comment]
Thanks @giladdi-tr!  If you have any insight into how this resource is charged those are always appreciated.  Or if you want to [contribute](https://github.com/infracost/infracost/blob/master/CONTRIBUTING.md) by writing some code we're happy to help.

**Comment by giladdi-tr** [comment]
@tim775 I will try to create a PR for it. (It is my first time doing it, I hope I will go smoothly)

**Comment by aliscott** [comment]
Thanks @giladdi-tr, please reach out on the [Infracost Slack](https://infracost.io/community-chat) or here for any help.

Our first step is working out how to map the prices to the Terraform resources. Do you know which [prices here](https://azure.microsoft.com/en-gb/pricing/details/service-bus/) you'd expect to see against `azurerm_servicebus_namespace` and which ones you'd expect to see against the `azurerm_servicebus_queue` or other resources?

**Comment by giladdi-tr** [comment]
@tim775 @aliscott, I see that estimating the cost of Azure Service Bus lower tiers is difficult. 
Service Basic and Standard tiers cost decided by the number of operations (operation = any API call). 
On the premium tier, the cost is running unit per hour (easy to estimate). 
What is your approach in those kinds of situations? 

This is the resource I expect to see cost estimation on: 
```
azurerm_servicebus_namespace
```

If I understood right, the other resources do not affect the cost. 
```
azurerm_servicebus_namespace_authorization_rule
azurerm_servicebus_namespace_disaster_recovery_config
azurerm_servicebus_namespace_network_rule_set
azurerm_servicebus_queue_authorization_rule
azurerm_servicebus_subscription
azurerm_servicebus_subscription_rule
azurerm_servicebus_topic
azurerm_servicebus_topic_authorization_rule
```

**Comment by aliscott** [comment]
Thanks @giladdi-tr this is the sort of insight that's super helpful for us!

For the basic and standard tier costs we can use the [Infracost usage file](https://www.infracost.io/docs/features/usage_based_resources/) to allow users to specify the number of operations. We've even started [experimenting with pulling the usage](https://www.infracost.io/docs/features/usage_based_resources/#fetch-from-cloudwatch) for the last 30 days directly from CloudWatch to populate that file. Do you think that would make sense?

**Comment by giladdi-tr** [comment]
@aliscott Very cooI. I will check it out. 
Will you also have a pulling usage feature on Azure?

**Comment by aliscott** [comment]
@giladdi-tr 👍 👍 yeah at the moment we're just testing it with AWS, but the idea would be that this works for AWS, Azure and GCP in the future.

**Comment by hugorut** [comment]
Support for `azurerm_servicebus_namespace` is now available as part of the [v0.10.22](https://github.com/infracost/infracost/releases/tag/v0.10.22) release.
